api/docker/migrations/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:12
+WORKDIR /usr/src/app
+
+#Installs sequelize and pg dependencies
+RUN npm install pg
+RUN npm install sequelize
+RUN npm install sequelize-cli
+
+#copies files needed for migrations
+COPY api/config/config.js /usr/src/app/api/config/config.js
```

Comment:
Can you point me to where these files are?

Reply:
I basically copied the structure of the platform__api and I added api/ on front, because all of my api code is in the [api folder](https://github.com/dev-launchers-sandbox/project__discord-bot/tree/api/api)

Reply:
The api branch will need to be merged first. We also need to add a github action to build the migration docker image https://github.com/dev-launchers/platform__api/blob/main/.github/workflows/migration-docker-build.yaml.